pkg/exporter/process_test.go

```diff
@@ -187,21 +207,30 @@ func TestExportingProcess_SendingDataRecordToLocalTCPServer(t *testing.T) {
 	}()
 
 	// Create exporter using local server info
-	exporter, err := InitExportingProcess(listener.Addr(), 1)
+	exporter, err := InitExportingProcess(listener.Addr(), 1, 0)
 	if err != nil {
 		t.Fatalf("Got error when connecting to local server %s: %v", listener.Addr().String(), err)
 	}
 	t.Logf("Created exporter connecting to local server with address: %s", listener.Addr().String())
 
-
-	// [Only for testing] Ensure corresponding template exists in the exporting process before sending data
-	templateID := exporter.AddTemplate()
-	// Hardcoding 8-bytes min data record length for testing purposes instead of creating template record
-	exporter.updateTemplate(templateID, &[]string{"sourceIPv4Address", "destinationIPv4Address"}, 8)
 	// Add data to exporting process
 	reg := registry.NewIanaRegistry()
 	reg.LoadRegistry()
 
+	// [Only for testing] Ensure corresponding template exists in the exporting process before sending data
```

Comment:
This line of comment can be deleted (I added it for the same purpose of line 231)

Reply:
Actually we still need as we are creating a template and updating template map. This is not required if we send a template record.

Reply:
Got it. Thanks